### ISSUE TITLE
Table Validator missing label for text input area

### ISSUE THREAD

**Issue by nschonni** [wet-boew/wet-boew#8328]
Running aXe on http://wet-boew.github.io/wet-boew/demos/tablevalidator/tablevalidator-en.html it flagged that `#inputsourcecode` is only using placeholder text and not a programmatic label
